src/EncumberableTokenFactory.sol

```diff
@@ -10,6 +10,9 @@ import { EncumberableToken } from "./EncumberableToken.sol";
  * @author Compound
  */
 contract EncumberableTokenFactory {
+    /// @dev Emitted when new wrapper is deployed for `underlyingToken`
+    event WrapperDeployed(address indexed underlyingToken, address indexed wrapperToken);
```

Comment:
nit: `DeployWrapper` seems more consistent with what we have done in the past with event names in Comet (e.g. `TransferCollateral`). That being said, I haven't found an established set of conventions for naming events

Reply:
updated to `DeployWrapper`